src/components/nav-bar/socials/Socials.js

```diff
@@ -19,7 +19,7 @@ const Socials = () => {
         </a>
       </SocialInnerWrapper>
       {/* <SocialInnerWrapper>
-        <a href="http://eh-fm.chatango.com/" target="blank">
+        <a href="https://eh-fm.chatango.com/" target="blank">
           <NavChat src={ChatangoLogo} alt="chatango" />
         </a>
       </SocialInnerWrapper> */}
```

Comment:
Can probably just be killed?

Reply:
[Removed](https://github.com/RichardPK/eh-fm/pull/45/commits/6f074cd5ee5ec1d2b1672b91f75f9714bb70f583)